src/tbbmalloc/frontend.cpp

```diff
@@ -1986,7 +1986,7 @@ class ShutdownSync {
     }
     void processExit() {
         if (flag.fetch_add(skipDtor) != 0) {
-            SpinWaitUntilEq(flag, skipDtor);
+            tbb::detail::spin_wait_until_eq(flag, skipDtor);
```

Comment:
It does not seems as a good approach because it brings uncontrolled dependency on TBB